src/components/Capture/capture.js

```diff
@@ -233,22 +245,27 @@ const CaptureMode = impurify(({method, side, useCapture, ...other}) => (
     <Title {...{method, side, useCapture}}/>
     {useCapture ?
       <Camera {...{method, ...other}}/> :
-      <Uploader {...{method,...other}}/>
+      <Uploader {...{method, ...other}}/>
     }
   </div>
 ))
 
-const CaptureScreen = ({method, side, validCaptures, useCapture, advancedValidation, token, uploadInProgress, onApiUpload, onApiError, ...other}) => {
+const CaptureScreen = ({validCaptures, useCapture, uploadInProgress, ...other}) => {
   const hasCapture = validCaptures.length > 0
-  return (<div className={classNames({
-    [style.camera]: useCapture && !hasCapture,
-    [style.uploader]: !useCapture && !hasCapture
-  })}>
-    {hasCapture ?
-      uploadInProgress ? <ProcessingApiRequest /> : <Confirm {...{ method, side, validCaptures, token, advancedValidation, onApiUpload, onApiError, ...other}} /> :
-      <CaptureMode {...{method, side, useCapture, ...other}} />
+  return (
+    <div
+      className={classNames({
+        [style.camera]: useCapture && !hasCapture,
+        [style.uploader]: !useCapture && !hasCapture})}
+    >
+    { uploadInProgress ?
```

Comment:
yeah, not the prettiest, I still think the nesting is better than the way it was before. @stefaniacardenas thoughts?